.env.example

```diff
@@ -1,5 +1,5 @@
 API_URL=http://localhost:3000
 PORT=3000
-DB_CONNECTION_STRING=''
+DB_CONNECTION_STRING=postgresql://obox_admin:eomkyby1nvXktZC5U7fLlfShLh9S64cA@dpg-cprhs6bqf0us738fmp30-a.frankfurt-postgres.render.com/obox_fpqx?ssl=true
```

Comment:
this is sensitive, things like that should never be pushed to repository. store it in your local .env file only